_includes/post_import.html

```diff
@@ -1,7 +1,7 @@
 {% if page.odfeimport %}
 <div class="imported-post">
-    This post was imported from the <i>Open Distro For Elasticsearch</i> blog, 
-    a predecessor project of OpenSearch. Information reflected in this post
-     may not be current or accurate.
+    This post was imported from the <i>Babelfish for PostgreSQL</i> blog, 
+    a Postgres extension for handling a set of MS SQL Server capabilities. 
+    Information reflected in this post may not be current or accurate.
```

Comment:
How about:

This post was imported from the <i>Babelfish for PostgreSQL</i> blog. Babelfish is a PostgreSQL extension that supports SQL Server SQL syntax on a PostgreSQL server. Information in this post may not be current or accurate.